pkgs/applications/networking/cluster/kfilt/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, buildGo120Module, fetchFromGitHub }:
```

```suggestion
{ lib, buildGoModule, fetchFromGitHub }:
```

`stdenv` isn't used + suggestion below/above regarding `buildGoModule`